jenkinsbuild.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Parse arguments.
+while [[ $# -gt 1 ]]
+do
+key="$1"
+case $key in
+    -s|--server-name)
+    SERVERNAME="$2"
+    shift
+    ;;
+    -t|--target)
+    TARGET="$2"
+    shift
+    ;;
+    -c|--compiler)
+    CCOMP="$2"
+    shift
+    ;;
+    -cxx|--cxx-compiler)
+    CXXCOMP="$2"
+    shift
+    ;;
+    -m|--compile-mode)
+    COMPILEMODE="$2"
+    shift
+    ;;
+    -n|--build-number)
+    BUILDID="$2"
+    shift
+    ;;
+    -b|--branch)
+    BRANCH="$2"
+    shift
+    ;;
+    -32|--force-32)
+    FORCE32="$2"
+    shift
+    ;;
+    *)
+    ;;
+esac
+shift
+done
+
+git submodule update --init
+
+# Set up build information.
+export CUBERITE_BUILD_SERIES_NAME="$SERVERNAME $TARGET $COMPILEMODE ($BRANCH)"
+export CUBERITE_BUILD_ID="$BUILDID"
+export CUBERITE_BUILD_DATETIME="`date`"
+
+export CCACHE_CPP2=true
+CACHE_ARGS="-DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache"
+
+# Build
+CXX=$CXXCOMP CC=$CCOMP cmake . -DNO_NATIVE_OPTIMIZATION=1 -DBUILD_TOOLS=1 -DCMAKE_BUILD_TYPE=${COMPILEMODE^^} -DFORCE_32=${FORCE32^^}
+make -j 2
+
+# Prepare artifacts directory
+mkdir artifacts
```

Comment:
This broke existing download links (and scripts that depend on them).
Old: https://builds.cuberite.org/job/Cuberite%20Linux%20x64%20Master/lastSuccessfulBuild/artifact/Cuberite.tar.gz
New: https://builds.cuberite.org/job/Cuberite%20Linux%20x64%20Master/lastSuccessfulBuild/artifact/artifacts/Cuberite.tar.gz

Reply:
Unfortunately it did - the Jenkins UI hid the change to the URL and I didn't think to check it manually :( 

I have reverted this change and I will create a proper canonical link for future use that just redirects to the current download location; but for the forseeable future we have to stick with the current location.